scalability_and_performance/what-huge-pages-do-and-how-they-are-consumed-by-apps.adoc

```diff
@@ -9,4 +9,13 @@ include::modules/what-huge-pages-do.adoc[leveloffset=+1]
 
 include::modules/how-huge-pages-are-consumed-by-apps.adoc[leveloffset=+1]
 
+include::modules/consuming-huge-pages-resource-using-the-downward-api.adoc[leveloffset=+1]
+
+// Info about Downward API is in the node-containers area.
+// Seems important to make readers aware that it exists.
+// Is this block title part of the preceding module? Yes it is. Squint a little.
+.Additional resources
+
+* xref:../nodes/containers/nodes-containers-downward-api.adoc[Allowing containers to consume Downward API objects]
```

Comment:
.adoc#nodes-containers-downward-api[Allowing containers to consume Downward API objects]

You must specify an anchor ID per https://github.com/openshift/openshift-docs/blob/master/contributing_to_docs/doc_guidelines.adoc#links-to-internal-content